### PR TITLE
Cleaning paws

### PR DESCRIPTION
**Pull request by VictorSanh**

### CODE REVIEW

File: promptsource/templates/paws-x/en/templates.yaml [hunk 1]
```diff
@@ -1,203 +1,205 @@
...
-      choices_in_prompt: null
-      metrics: []
-      original_task: false
-    name: paraphrase-task-reverse
-    reference: Create a generative paraphrase task
-  2ff509a8-2712-4406-97bd-ce86824cdc18: !Template
-    answer_choices: null
+  0be7cecd-b427-4ec9-9b0e-666d6dae00dd: !Template
+    answer_choices:
+    - 'No'
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:6]: @stephenbach changed the answer choices format so you can write this as  "No ||| Yes" now. Stephen are we keeping the old format as well?

File: promptsource/templates/paws-x/en/templates.yaml [hunk 2]
```diff
@@ -1,203 +1,205 @@
...
+    - 'Yes'
     answer_choices_key: null
-    id: 2ff509a8-2712-4406-97bd-ce86824cdc18
-    jinja: "{% if label == 0 or label == 1 %} \n{{sentence1}} Question: {{sentence2}}\
-      \ paraphrase or not?\n||| \n{% if label == 0 %} \nFalse\n{% elif label == 1\
-      \ %}\nTrue\n{% endif %}\n{% endif %}"
+    id: 0be7cecd-b427-4ec9-9b0e-666d6dae00dd
+    jinja: "Determine if the following two sentences paraphrase or not.\nSent 1: {{sentence1}}\n\
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:10]: "paraphrase each other"

File: promptsource/templates/paws-x/en/templates.yaml [hunk 3]
```diff
@@ -1,203 +1,205 @@
...
+    - 'No'
+    - 'Yes'
     answer_choices_key: null
-    id: 568abe6e-4014-4d20-b4a0-3b0622026e36
-    jinja: "{% if label == 0 or label == 1 %} \nSentence 1: {{sentence1}}\\nSentence\
-      \ 2: {{sentence2}}\\nQuestion: Does Sentence 1 and Sentence 2 express same meaning\
-      \ or not\n||| \n{% if label == 0 %} \nNo\n{% elif label == 1 %}\nYes\n{% endif\
-      \ %}\n{% endif %}"
+    id: 472fe5eb-b499-4952-a930-f72f4ca9eddd
+    jinja: "Sentence 1: {{sentence1}}\nSentence 2: {{sentence2}}\nQuestion: Does Sentence\
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:27]: "Do Sentence 1 and Sentence 2"

File: promptsource/templates/paws-x/en/templates.yaml [hunk 4]
```diff
@@ -1,203 +1,205 @@
...
+    - 'Yes'
     answer_choices_key: null
-    id: 68f76857-66f0-4aa4-ad70-3443cce0c7ae
-    jinja: "{% if label == 0 or label == 1 %} \nSentence 1: {{sentence1}}\\nSentence\
-      \ 2: {{sentence2}}\\nQuestion: Can we rewrite Sentence 1 by Sentence 2? \n|||\
-      \ \n{% if label == 0 %} \nNo\n{% elif label == 1 %}\nYes\n{% endif %}\n{% endif\
-      \ %}"
+    id: 678400f8-1a5c-4a40-b5ef-abeaa41e20dd
+    jinja: "Sentence 1: {{sentence1}}\nSentence 2: {{sentence2}}\nQuestion: Can we\
+      \ rewrite Sentence 1 by Sentence 2? \n||| \n{{answer_choices[label]}}"
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:62]: Small "by" => "to"

File: promptsource/templates/paws-x/en/templates.yaml [hunk 5]
```diff
@@ -1,203 +1,205 @@
...
+    - 'Yes'
     answer_choices_key: null
-    id: c7e71c01-ea6d-4cff-ad2d-c05d4a7ce9ad
-    jinja: "{% if label == 0 or label == 1 %} \nSentence 1: {{sentence1}}\\nSentence\
-      \ 2: {{sentence2}}\\nQuestion: Does Sentence 1 paraphrase Sentence 2? Yes or\
-      \ No? \n||| \n{% if label == 0 %} \nNo\n{% elif label == 1 %}\nYes\n{% endif\
-      \ %}\n{% endif %}"
+    id: d5239f5f-2014-47c9-a0c1-4896f76f82dd
+    jinja: "Sentence 1: {{sentence1}}\nSentence 2: {{sentence2}}\nQuestion: Does Sentence\
+      \ 1 and Sentence 2 express the same meaning? \n||| \n{{answer_choices[label]}}"
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:146]: "Does" => "Do"

File: promptsource/templates/paws-x/en/templates.yaml [hunk 6]
```diff
@@ -1,203 +1,205 @@
...
+      - Accuracy
       original_task: true
-    name: Concatenation
-    reference: Concatenation of sentence 1 and sentence 2
-  c873732e-c424-417f-a474-f19eff2caf35: !Template
-    answer_choices: null
+    name: Meaning-no-label
+    reference: Natural question without label
+  d9911dad-75fe-4506-9843-3a46ba5e49dd: !Template
+    answer_choices:
```
Comment (srush) [inline promptsource/templates/paws-x/en/templates.yaml:157]: As above

File: promptsource/templates/paws/labeled_final/templates.yaml [hunk 7]
```diff
@@ -2,95 +2,104 @@ dataset: paws
...
-    answer_choices: null
+    answer_choices:
+    - 'No'
+    - 'Yes'
     answer_choices_key: null
     id: 0be7cecd-b427-4ec9-9b0e-666d6dae0063
-    jinja: "{% if label == 0 or label == 1 %} \nDetermine if the following two sentences\
-      \ paraphrase or not.\n{{sentence1}}\n{{sentence2}}\n||| \n{% if label == 0 %}\
-      \ \nNo\n{% elif label == 1 %}\nYes\n{% endif %}\n{% endif %}"
+    jinja: "Determine if the following two sentences paraphrase or not.\nSent 1: {{sentence1}}\n\
```
Comment (srush) [inline promptsource/templates/paws/labeled_final/templates.yaml:10]: "Paraphrases"

File: promptsource/templates/paws/labeled_swap/templates.yaml [hunk 8]
```diff
@@ -1,203 +1,205 @@
...
+    answer_choices:
+    - 'No'
+    - 'Yes'
     answer_choices_key: null
-    id: 170f7d5a-096e-4caf-abee-afd74918783b
-    jinja: "{% if label == 0 or label == 1 %} \nDetermine if the following two sentences\
-      \ paraphrase or not.\n{{sentence1}}\n{{sentence2}}\n||| \n{% if label == 0 %}\
-      \ \nNo\n{% elif label == 1 %}\nYes\n{% endif %}\n{% endif %}"
+    id: 0be7cecd-b427-4ec9-9b0e-666d6dae00aa
+    jinja: "Determine if the following two sentences paraphrase or not.\nSent 1: {{sentence1}}\n\
```
Comment (srush) [inline promptsource/templates/paws/labeled_swap/templates.yaml:10]: As above.

**Review by srush — CHANGES_REQUESTED** [review]
Some grammar changes. I stopped midway as there were a lot of repeats.

File: promptsource/templates/paws-x/en/templates.yaml (hunk 1 above)
Comment (VictorSanh) [reply promptsource/templates/paws-x/en/templates.yaml:6]: i thought we were merging all `answer_choices` and `answer_choices_key` for all templates later

**Review by srush — APPROVED** [review]

**Pull request merged**